CHANGES.rst

```diff
@@ -2,7 +2,12 @@ Changelog
 =========
 
 
-- Fixes pytest-xdist 2.0 compatibility (olegpidsadnyi) #369.
+- Strict Gherkin option is removed. (olegpidsadnyi)
+- Given step is no longer a fixture. The scope parameter is also removed. (olegpidsadnyi)
```

Comment:
Let's leave a link to the "how to migrate from v3.0.0" anchor

Reply:
done